Sources/Paywalls/Events/Networking/PaywallHTTPRequestPath.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PaywallHTTPRequestPath.swift
+//
+//  Created by Nacho Soto on 9/5/23.
+
+import Foundation
+
+extension HTTPRequest.PaywallPath: HTTPRequestPath {
+
+    // TODO: TBD
+    static let serverHostURL = URL(string: "https://api-paywalls.revenuecat.com")!
```

Comment:
@joshdholtz let me know when we know the final domain :)

Reply:
It will _probably_ be this but will confirm once its actually created 😛